### ISSUE TITLE
Profile: Create new pages to support profile updates

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/Help#80]
Create new pages to support Profile > Eligibility and Profile > Demographics

* [x] /Help/how-to/account/profile/eligibility/military-service/
* [x] /Help/how-to/account/profile/eligibility/federal-employment/

**Comment by humancompanion** [comment]
Just pushed a change to stub out all of these pages. Need to work with @dalenard21 to come up with a good format for explaining question options and not repeating other sections.

**Comment by dalenard21** [comment]
@humancompanion 

I have attached drafts of each of these.  I'm a bit unsure about this approach. 

**Pros**

- Keeps all help content about a specific section of "Profile" in one place.
- Is specific to the questions in Profile

**Cons**

- It is repetitive with content that already exists on the site
- Need to maintain two instances of similar/same content
- In some instances we link to other pages in help center for more information and in other instances we link over to FedHireVets or OPM. It's a bit inconsistent.

[Profile-eligibility-unique-hiring-path_20160621.docx](https://github.com/USAJOBS/Help/files/326386/Profile-eligibility-unique-hiring-path_20160621.docx)
[Profile-eligibility-federal-employment-status_20160621.docx](https://github.com/USAJOBS/Help/files/326387/Profile-eligibility-federal-employment-status_20160621.docx)
[Profile-eligibility-military-service_20160621.docx](https://github.com/USAJOBS/Help/files/326388/Profile-eligibility-military-service_20160621.docx)

**Comment by dalenard21** [comment]
@humancompanion

I have an updated military service doc as well as a few edits to the unique hiring path doc.
[Profile-eligibility-unique-hiring-path_20160621.docx](https://github.com/USAJOBS/Help/files/335603/Profile-eligibility-unique-hiring-path_20160621.docx)
[Profile-eligibility-military-service_20160621.docx](https://github.com/USAJOBS/Help/files/335604/Profile-eligibility-military-service_20160621.docx)



I'll run these by Peter and Tracy, but let's use these as placeholder content.

**Comment by humancompanion** [comment]
@dalenard21 and I decided to kill the /Help/how-to/account/profile/demographics/ page.

**Comment by humancompanion** [comment]
@dalenard21 - I've verified that the military service and federal employment pages are up-to-date. Where should I drop in this unique-hiring-path content ("How to determine if you’re a unique hiring path")? Should that go beneath (or I suppose above is an option) the current content?:

http://usajobs.github.io/Help/working-in-government/unique-hiring-paths/

I believe we decided that the link in Profile > Preferences > Unique Hiring Paths was going to the link above rather than a how-to. Am I recalling our conversation correctly? Thanks.
